examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -109,7 +117,6 @@
     score_iterative_imputer[impute_estimator.__class__.__name__] = cross_val_score(
         estimator, X_missing, y_missing, scoring="neg_mean_squared_error", cv=N_SPLITS
     )
-
```

Comment:
please revert such changes.

Reply:
The only reason I selected 3 fold was BayesianRidge! Between All Iterative Imputers in 3 and 5 folds best performance (OOF Score) was from Iterative BasianRidge in 3 folds, With MSE 0.676844.

Reply:
That may just be an accident, 3 is just not a large enough number for any CV

Reply:
It's almost funny. For different random state BayesianRidge with 3 folds on this dataset performs better than original Data!
Original          Full Data                0.657900
SimpleImputer     mean                     0.862643
                  median                   0.867845
IterativeImputer  BayesianRidge            0.613065
                  DecisionTreeRegressor    0.789176
                  ExtraTreesRegressor      0.746601
                  KNeighborsRegressor      0.866287